src/helpers/ItemList.js

```diff
@@ -0,0 +1,202 @@
+    description:
+      "A buttery, flakey croissant with sweet, almond filling and topped with sliced almonds.",
+    type: "bakedGood",
+  },
+  {
+    name: "House Brewed Coffee",
+    image: brewedCoffee,
+    description:
+      "Our light roasted house brew, brewed from only the top 1% of Arabica beans in the world.",
+    type: "brewedCoffee",
+  },
+  {
+    name: "Café Au Lait",
+    image: cafeAuLait,
+    description:
+      "Our light roast coffee with steamed whole milk and topped with thick velvety foam.",
+    type: "brewedCoffee",
+  },
+  {
+    name: "Café Caramel",
+    image: cafeCaramel,
+    description:
+      "Our light roast coffee with French vanilla powder, caramel sauce, steamed non-fat milk, topped with thick foam, and a drizzle of caramel sauce.",
+    type: "brewedCoffee",
+  },
+  {
+    name: "Café Hazelnut",
+    image: cafeHazelnut,
+    description:
+      "Our light roast coffee with hazelnut powder, steamed non-fat milk and topped with thick foam.",
+    type: "brewedCoffee",
+  },
+  {
+    name: "Café Mocha",
+    image: cafeMocha,
+    description:
+      "This signature drink combines our light roast coffee with Dutch chocolate powder, steamed non-fat milk and topped with thick foam.",
+    type: "brewedCoffee",
+  },
+  {
+    name: "House Iced Coffee",
+    image: icedCoffee,
+    description:
+      "Our specially brewed coffee served over ice for a refreshing and bold coffee taste.",
+    type: "icedCoffee",
+  },
+  {
+    name: "Caramel Iced Coffee",
+    image: icedCoffeeCaramel,
+    description:
+      "Our specially brewed coffee served over ice for a refreshing and bold coffee taste.",
+    type: "icedCoffee",
+  },
+  {
+    name: "Hazelnut Iced Coffee",
+    image: icedCoffeeHazelnut,
+    description:
+      "Our premium espresso shots blended with our specially developed hazelnut powder and served over ice for a refreshing and delicious hazelnut coffee drink.",
+    type: "icedCoffee",
+  },
+  {
+    name: "Mocha Iced Coffee",
+    image: icedCoffeeMocha,
+    description:
+      "Our premium espresso shots blended with Dutch chocolate powder and served over ice for a refreshing and delicious chocolate coffee drink.",
+    type: "icedCoffee",
+  },
+  {
+    name: "Vanilla Iced Coffee",
+    image: icedCoffeeVanilla,
+    description:
+      "Our premium espresso shots blended with our French Deluxe vanilla powder and served over ice for a refreshing and delicious vanilla coffee drink",
+    type: "icedCoffee",
+  },
+];
+
+export const itemList2 = itemList.map((obj) => {
+  let price;
+
+  switch (obj.type) {
+    case "donut":
+      price = 3;
+      break;
+    case "bakedGood":
+      price = 4;
+      break;
+    case "brewedCoffee":
+      price = 5;
+      break;
+    case "icedCoffee":
+      price = 6;
+      break;
+    default:
+      price = 1;
+      break;
+  }
+  return {
+    ...obj,
+    price,
+    inventory: 10,
+  };
+});
+
+console.log("is this working", itemList2);
```

Comment:
If it this working, let's avoid keeping any console.logs

Reply:
console.log removed.